openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -134,24 +137,21 @@ def award_program_certificates(self, username):
     """
     LOGGER.info('Running task award_program_certificates for username %s', username)
 
-    config = ProgramsApiConfig.current()
     countdown = 2 ** self.request.retries
 
-    # If either programs or credentials config models are disabled for this
+    # If the credentials config model is disabled for this
     # feature, it may indicate a condition where processing of such tasks
     # has been temporarily disabled.  Since this is a recoverable situation,
     # mark this task for retry instead of failing it altogether.
-    if not config.is_certification_enabled:
-        LOGGER.warning(
-            'Task award_program_certificates cannot be executed when program certification is disabled in API config',
-        )
-        raise self.retry(countdown=countdown, max_retries=config.max_retries)
 
-    if not CredentialsApiConfig.current().is_learner_issuance_enabled:
+    credentials_config = CredentialsApiConfig.current()
+    credentials_config.version = 2
```

Comment:
Ditch the property. Use the URL. I would add a migration disabling the functionality. This will force us to manually update the URL and reactivate the feature. We don't want a migration setting/modifying the URL, because we aren't the only consumers of this code.